packages/build-info/src/frameworks/framework.ts

```diff
@@ -140,11 +142,26 @@ export function sortFrameworksBasedOnAccuracy(a: DetectedFramework, b: DetectedF
   return sort
 }
 
-/** Merges a list of detection results based on accuracy to get the one with the highest accuracy */
+/** Merges a list of detection results based on accuracy to get the one with the highest accuracy that still contains information provided by all other detections */
 export function mergeDetections(detections: Array<Detection | undefined>): Detection | undefined {
-  return detections
-    .filter(Boolean)
-    .sort((a: Detection, b: Detection) => (a.accuracy > b.accuracy ? -1 : a.accuracy < b.accuracy ? 1 : 0))?.[0]
+  const definedDetections = detections
+    .filter(function isDetection(d): d is Detection {
+      return Boolean(d)
+    })
+    .sort((a: Detection, b: Detection) => (a.accuracy > b.accuracy ? -1 : a.accuracy < b.accuracy ? 1 : 0))
+
+  if (definedDetections.length === 0) {
+    return
+  }
+
+  return definedDetections.slice(1).reduce((merged, detection) => {
```

Comment:
The way I read this block is merge the first two detections, prioritizing the first item in the list. Your comment edit at the top says "...that still contains information provided by all other detections." Just want to make sure the comment and the code are aligned is all.

Reply:
`.slice(1)` will give you previous array with 1st item removed - I do this, because I use that first item as starting point on to which I will be merging information from remaining items

Reply:
Ah sorry, I seem to be a bit rusty 😖 Makes perfect sense!